distribution/LICENSE-server

```diff
@@ -0,0 +1,444 @@
+
+    org.codehaus.janino:commons-compiler 3.1.10 BSD-3-Clause see:licenses/janino-BSD-3-Clause
+    org.codehaus.janino:janino 3.1.10 BSD-3-Clause see:licenses/janino-BSD-3-Clause
+    com.thoughtworks.xstream:xstream 1.4.21 BSD-3-Clause see:licenses/xstream-BSD-3-Clause
+    org.ow2.asm:asm 6.0 BSD-3-Clause see:licenses/asm-BSD-3-Clause
+    org.hamcrest:hamcrest 2.2 BSD-3-Clause see:licenses/hamcrest-BSD-3-Clause
+    org.hamcrest:hamcrest-core 2.2 BSD-3-Clause see:licenses/hamcrest-BSD-3-Clause
+    com.esotericsoftware:kryo 5.4.0 BSD-3-Clause see:licenses/kryo-BSD-3-Clause
+    com.esotericsoftware:minlog 1.3.1 BSD-3-Clause see:licenses/minlog-BSD-3-Clause
+    com.esotericsoftware:reflectasm 1.11.9 BSD-3-Clause see:licenses/reflectasm-BSD-3-Clause
+    com.google.protobuf:protobuf-java 3.25.5 BSD-3-Clause see:licenses/protobuf-java-BSD-3-Clause
+    com.google.protobuf:protobuf-java-util 3.11.0 BSD-3-Clause see:licenses/protobuf-java-BSD-3-Clause
+
+========================================================================
+CDDL-1.0 licenses
+========================================================================
+
+    com.sun.jersey.contribs:jersey-apache-client4 1.19.1 CDDL-1.0 see:licenses/CDDL-1.0
+    com.sun.jersey:jersey-client 1.19.1 CDDL-1.0 see:licenses/CDDL-1.0
+    com.sun.jersey:jersey-core 1.19.1 CDDL-1.0 see:licenses/CDDL-1.0
+    javax.ws.rs:jsr311-api 1.1.1 CDDL-1.0 see:licenses/CDDL-1.0
+
+========================================================================
+EPL-1.0 licenses
+========================================================================
+
+    ch.qos.logback:logback-classic 1.2.12 EPL-1.0 see:licenses/EPL-1.0
+    ch.qos.logback:logback-core 1.2.12 EPL-1.0 see:licenses/EPL-1.0
+    junit:junit 4.13.2 EPL-1.0 see:licenses/junit4-EPL-1.0
+
+========================================================================
+EPL-2.0 licenses
+========================================================================
+
+    jakarta.annotation:jakarta.annotation-api 1.3.5 EPL-2.0 see:licenses/EPL-2.0
+
+========================================================================
+MIT licenses
+========================================================================
+
+    org.checkerframework:checker-qual 3.37.0 MIT see:licenses/checker-qual-MIT
+    redis.clients:jedis 3.8.0 MIT see:licenses/jedis-MIT
+    org.slf4j:jul-to-slf4j 1.7.36 MIT see:licenses/jul-to-slf4j-MIT
+    org.slf4j:slf4j-api 1.7.36 MIT see:licenses/slf4j-api-MIT
+    com.github.andrewoma.dexx:dexx-collections 0.2 MIT see:licenses/dexx-collections-MIT
+
+========================================================================
+Public Domain licenses
+========================================================================
+
+    aopalliance:aopalliance 1.0 Public Domain
+
+========================================================================
+Python-2.0 licenses
+========================================================================
```

Comment:
Remove this dependency. This is a build tool and there is no source code dependency.